src/metabase/lib/schema/expression.cljc

```diff
@@ -172,7 +172,15 @@
 (mr/def ::expression
   [:maybe (expression-schema :type/* "any type of expression")])
 
+(mr/def ::expression.definition
+  [:and
+   [:ref ::expression]
+   [:cat
+    #_tag :any
+    #_opts [:map
+            [:lib/expression-name [:string {:decode/normalize common/normalize-string-key}]]]
```

Comment:
We were missing normalization behavior for the expression name. Not sure why it would ever be a keyword but I ran into someone doing it somewhere